tests/models/nn/transformers/test_base.py

```diff
@@ -152,10 +153,24 @@ def test_save_load_for_fitted_model(
 
     @pytest.mark.parametrize("test_dataset", ("dataset", "dataset_item_features"))
     @pytest.mark.parametrize("model_cls", (SASRecModel, BERT4RecModel))
+    @pytest.mark.parametrize("map_location", ("cpu", torch.device("cuda:0"), None))
+    @pytest.mark.parametrize(
```

Comment:
You are not iterating over variants here so it shouldn't be parametrized. The correct way is to specify None as a second option